cmd/create/release/runner_test.go

```diff
@@ -21,3 +25,46 @@ func Test_findNewRelease(t *testing.T) {
 		t.Errorf("expected aws, found %s", provider)
 	}
 }
+
+func Test_generateReleaseName(t *testing.T) {
+	t.Skip("timing sensitive tests; only enable on-demand locally")
```

Comment:
These tests are very timing specific due to nature of release name generation. I felt that was good enough and this test is mostly useful when locally verifying the generation function so I don't feel too bad skipping it in general. WDYT?